src/tests/test-grpc-server.sh

```diff
@@ -25,10 +26,12 @@ lua=${5:-lua5.4}
 server_address=127.0.0.1:5001
 checkin_address=127.0.0.1:5002
 
+export LUA_PATH_5_4="$( dirname "${script_dir}")/?.lua;${LUA_PATH_5_4}"
```

Comment:
You should use `${LUA_PATH_5_4:-;}` instead of `${LUA_PATH_5_4}`, otherwise in case `LUA_PATH_5_4` is not properly set in the environment, the scripts will be at risk of not finding system wide Lua libraries. This can happen in some systems.

`;${LUA_PATH_5_4:-;}` is a trick to evaluate to `;;` when `LUA_PATH_5_4` is unset, in this case Lua will use system wide search paths.